src/walletSdk/index.ts

```diff
@@ -1,18 +1,115 @@
+import StellarSdk, { Networks, Server } from "stellar-sdk";
+
 import { Anchor } from "./anchor";
+import { WalletSigner, DefaultSigner } from "./auth/WalletSigner";
+import { Stellar } from "./horizon/Stellar";
+import { NETWORK_URLS } from "./horizon/constants";
+import { Recovery } from "./recovery/Recovery";
+import { getUrlDomain } from "./util/url";
+
+// TODO - https://stellarorg.atlassian.net/browse/WAL-789?atlOrigin=eyJpIjoiZjcwNzZhOTJmNjE1NGRhNTk1NDlkOTExMTYxMDJkZmYiLCJwIjoiaiJ9
+interface ClientConfigFn {}
+interface ClientConfig {}
+class HttpClient {}
 
-export enum NETWORKS {
-  PUBLIC = "PUBLIC",
-  TESTNET = "TESTNET",
+export class Config {
```

Comment:
This is an internal class in kt, not sure if it should be exported

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/5/commits/fa22147df8a55a2e01b84faeb3bf578819a0b0ef